analyzer.go

```diff
@@ -48,34 +49,29 @@ func (t *Target) ModTime() (time.Time, error) {
 }
 
 type Analyzer struct {
-	values   map[string]Value
+	values   context.Context
```

Comment:
imho, using ctx in such way is anti-pattern
the code became more dirty

why do not just use the custom type over map?